### PR TITLE
doc: Fix the links to the asciidoc format

### PR DESCRIPTION
**Pull request by jpereira**
add .

### CODE REVIEW

File: doc/antora/modules/developers/pages/contributing.adoc [hunk 1]
```diff
@@ -90,10 +90,10 @@ diff -urN freeradiusd-version $MYSRC > ~/feature-version.patch
 === 2. Select e-mail destination
 
 If you are on the developers mailing list, send the patch there.
-<<mailto:freeradius-devel@lists.freeradius.org.adoc#,freeradius-devel@lists.freeradius.org>>
+freeradius-devel@lists.freeradius.org.adoc
```
Comment (arr2036) [inline doc/antora/modules/developers/pages/contributing.adoc]: mailto links should be replaced with ``link:mailto:<email address>[<email address>]

File: doc/antora/modules/howto/pages/INSTALL.adoc [hunk 2]
```diff
@@ -12,7 +12,7 @@ source. i.e. a ``tar.gz'' file.
 
 === Building from Source
 
-The <<../source/dependencies.adoc#,build dependencies>> must be
+The link:../source/dependencies.adoc[build dependencies] must be
```
Comment (arr2036) [inline doc/antora/modules/howto/pages/INSTALL.adoc]: The correct link here would be ``xref:developers:dependencies.adoc``

File: doc/antora/modules/howto/pages/INSTALL.adoc [hunk 3]
```diff
@@ -95,7 +95,7 @@ major versions of the server. (For example - version 2 to version 3, or
 version 3 to version 4.
 
 For details on what has changed between the version, see the
-<<../upgrade/#,upgade>> guide.
+link:../upgrade/[upgade] guide.
```
Comment (arr2036) [inline doc/antora/modules/howto/pages/INSTALL.adoc]: the correct link here would be xref:upgrade:index.adoc

File: doc/antora/modules/raddb/pages/README.adoc [hunk 4]
```diff
@@ -55,9 +55,9 @@ to make massive changes, and then to spend weeks debugging it.
 The files in this directory are organized into logical groups, as
 follows.
 
-* `mods-available/` - <<mods-available/README.adoc#,Available modules>>, with example configuration.
+* `mods-available/` - link:mods-available/README.adoc[Available modules], with example configuration.
 * `mods-enabled/` - Enabled modules that are being used by FreeRADIUS.
-* `sites-available/` - <<sites-available/README.adoc#,Available virtual servers>>, with example configuration.
+* `sites-available/` - link:sites-available/README.adoc[Available virtual servers], with example configuration.
```
Comment (arr2036) [inline doc/antora/modules/raddb/pages/README.adoc]: The correct link here would be ``xref:raddb:sites-available/index.adoc``

File: doc/antora/modules/raddb/pages/README.adoc [hunk 5]
```diff
@@ -134,7 +134,7 @@ Note that unlike v3, there are no longer any `authorize`,
 been renamed. This change was necessary in order to simplify the
 operation of the server, and to better support multiple protocols.
 
-Please see the <<../upgrade/#,upgrade>> file for more information
+Please see the link:../upgrade/[upgrade] file for more information
 about these changes.
```
Comment (arr2036) [inline doc/antora/modules/raddb/pages/README.adoc]: The correct link here would be ``xref:upgrade:index.adoc``

File: doc/antora/modules/raddb/pages/README.adoc [hunk 6]
```diff
@@ -162,12 +162,12 @@ This directory contains certificates and configuration for EAP and RADIUS over T
...
-* <<trigger.conf.adoc#,trigger conf>>
+* link:clients.conf.adoc[clients conf]
+* link:debug.conf.adoc[debug conf]
+* link:dictionary.adoc[dictionary]
+* link:experimental.conf.adoc[experimental conf]
+* link:panic.gdb.adoc[panic gdb]
+* link:radiusd.conf.adoc[radiusd conf]
+* link:radrelay.conf.adoc[radrelay conf]
+* link:templates.conf.adoc[templates conf]
+* link:trigger.conf.adoc[trigger conf]
```
Comment (arr2036) [inline doc/antora/modules/raddb/pages/README.adoc]: `s/link:/xref:/`

File: doc/antora/modules/unlang/pages/xlat/predefined.adoc [hunk 7]
```diff
@@ -780,7 +780,7 @@ Return named subcapture value from the last regular expression evaluated.
 
 Results of named capture groups are available using the `%{regex:<named capture
 group>}` expansion. They will also be accessible using the numbered expansions
-described <<_0_32,above>>.
+described link:_0_32[above].
```
Comment (arr2036) [inline doc/antora/modules/unlang/pages/xlat/predefined.adoc]: That's wrong and will break the in page linking.  The correct syntax for anchor links is ``<< >>``

File: doc/antora/modules/upgrade/pages/index.adoc [hunk 8]
```diff
@@ -120,7 +120,7 @@ For `udp`, the configuration entries are largley the same as for v3.
 e.g. `ipaddr`, `port`, etc.
 
 The `listen` section then compiles each
-<<Processing Sections,Processing Section>> based on the named packet
+link:Processing Sections[Processing Section] based on the named packet
```
Comment (arr2036) [inline doc/antora/modules/upgrade/pages/index.adoc]: Incorrect link, should be xref, refer to the actual file, and have an extension

File: doc/antora/modules/upgrade/pages/index.adoc [hunk 9]
```diff
@@ -139,7 +139,7 @@ This configuration is different from v3. The benefit is that it is much
 easier to understand. Instead of using things like
 `Post-Auth-Type Reject`, we now have just `send Access-Reject`.
 
-See also <<Processing Sections,Processing Sections>> for how the
+See also link:Processing Sections[Processing Sections] for how the
```
Comment (arr2036) [inline doc/antora/modules/upgrade/pages/index.adoc]: Same here

File: doc/antora/modules/unlang/pages/map.adoc [hunk 10]
```diff
@@ -23,7 +23,7 @@ database query which returns one or more fields.
 taken from the database query.
 
 <op>:: The operator such as `=`, `:=`, etc. Any
-<<update.adoc#_editing_operators,editing operator>> defined for an
+link:update.adoc#_editing_operators[editing operator] defined for an
```
Comment (arr2036) [inline doc/antora/modules/unlang/pages/map.adoc]: Should be xref not link

File: doc/antora/modules/unlang/pages/xlat/predefined.adoc (hunk 7 above)
Comment (arr2036) [reply doc/antora/modules/unlang/pages/xlat/predefined.adoc]: It's possible ``xref:predefined.adoc#_0_32[above]`` might work too.

File: doc/antora/modules/developers/pages/contributing.adoc [hunk 11]
```diff
@@ -4,7 +4,7 @@
 
 For a person or company wishing to submit a change to the FreeRADIUS
 project the process can sometimes be daunting if you’re not familiar
-with ``the system.'' This text is a collection of suggestions which can
+with `the system`. This text is a collection of suggestions which can
```
Comment (alandekok) [inline doc/antora/modules/developers/pages/contributing.adoc]: I don't see why this should be fixed-width.  It's not referring to a command, debug output, unlang, or anything else like that.

Just use double quotes.

File: doc/antora/modules/developers/pages/module_interface.adoc [hunk 12]
```diff
@@ -76,27 +76,18 @@ different stages. The components are:
...
-|accounting |Record the request in the log.
-
-|checksimul |Count the number of active sessions for the user.
-
-|postauth |Process the response before it’s sent to the NAS.
-
-|preproxy |Process a request before it’s proxied.
-
-|postproxy |Filter attributes from a reply to a proxied request.
+| Component      | Description
```
Comment (alandekok) [inline doc/antora/modules/developers/pages/module_interface.adoc]: There isn't a lot of point in reformatting this when it's entirely wrong for v4.

**Comment by jpereira** [comment]
@alandekok @arr2036 updated.

**Pull request merged**